CONTRIBUTING.md

```diff
@@ -352,7 +354,44 @@ In cases where a PR is not worth noting in the release notes, you can also tell
 
 Finally, don't fret about this too much! The Release Drafter configuration and labeling scheme may take some time to fine-tune, and the drafted release notes can always be manually edited before final publication.
 
-## Publishing to npm
+## npm package
+
+### Testing the package locally
```

Comment:
I suggest that this might be a better fit as a new subsection within "Getting started", called "Testing your changes as a dependency in another project". Thinking from the perspective of someone starting out on contributing to explorer-1, I feel like it would fit more naturally into the flow if it were up there. Thoughts?